threedi_modelchecker/simulation_templates/models.py

```diff
@@ -183,15 +181,98 @@ async def save_to_api(self, client: V3BetaApi, simulation: Simulation):
                     simulation_pk=simulation.id, data=self.constant_gw
                 )
             )
-        if self.predefined_1d is not None:
+        initial_waterlevels: List[InitialWaterlevel] = []
+        rasters_lookup: Dict = {}
+
+        # TODO: support user uploaded 1d/2d/gw initial waterlevel file/rasters to threedimodels.
```

Comment:
I don't think that is really possible; this module only reads sqlites and everything attached to that.

Reply:
You can create SimulationTemplate object programatically or from a dict.
I'm hoping that Lutra is also able to use this part (or maybe even also the sqlite extraction part) to create simulation templates.